### DISCUSSION TITLE
Automatic suggestion detection threshold

### DISCUSSION THREAD

**Discussion by tomkolp** [WeblateOrg/weblate#10933] [General] [answered]
Has the automatic suggestion threshold been changed recently?  Now most automatic suggestions are only 100%, rarely less.  We used these smaller thresholds a lot.  The longer the text, the worse it is.

**Comment by nijel** [comment]
Yes, the PostgreSQL similarity threshold calculation has been adjusted some time ago to do only closer matches:

https://github.com/WeblateOrg/weblate/blob/68df5a14d90b0a57356217e1bf4e5d38c7e92401/weblate/memory/models.py#L62-L81

If using automatic translation, you can configure the similarity threshold in it.

**Comment by tomkolp** [comment]
Currently it is 75 and should I change this value to a smaller one?  How long was it before?  

Do I need to make changes inside the weblate-docker-weblate-1 container?  Do you have to do it again after each update?  

Is it possible to enter these settings into the gui?  It would be great if you could change this value for a specific component.

**Reply by nijel** [reply, marked as answer]
There is no way to configure what "Automatic suggestions" tab does. But when using add-on or invoke automatic translation in the UI, you define your own threshold there.
